Gem/Code/Source/GameLift/GameLiftClientComponent.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Console/IConsole.h>
+#include <AzFramework/Matchmaking/MatchmakingNotifications.h>
+
+#include <Request/AWSGameLiftRequestBus.h>
+#include <Request/AWSGameLiftSessionRequestBus.h>
+#include <Request/AWSGameLiftMatchmakingRequestBus.h>
+
+
+namespace MultiplayerSample
+{
+    class GameLiftClientSystemComponent

```

Comment:
Consider calling this the MultiplayerGameLiftClientSystemComponent or MPSGameLiftClientSystemComponent to avoid any confusion with things in the GameLift gem.

Add comment about class description/responsibilities